Task6/src/main/java/arrays/ArrayListAndArray.java

```diff
@@ -0,0 +1,60 @@
+package arrays;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class ArrayListAndArray {
+    private String[] array =new String[10];
+    private ArrayList<String> list = new ArrayList<String>();
+    long timeArrayAdd100k;
+    long timeListAdd100k;
+
+    public void addStrings(String[] array, String newString){
```

Comment:
If you don't plan to use this method outside class then it should be private.
Moreover, general rule of thumb methods come in order: 1st: main public methods that invoked in other classes, 2nd: private auxiliary method which used in 1st type methods.
Please make it private and put at the buttom.

Reply:
done